tuxemon/technique.py

```diff
@@ -356,7 +402,7 @@ def apply_lifeleech(self, user: Monster, target: Monster) -> EffectResult:
 
         """
         already_applied = any(t for t in target.status if t.slug == "status_lifeleech")
-        success = not already_applied and self.can_apply_status and self.potency >= random.random()
```

Comment:
why was can_apply_status removed?

Reply:
as i said above, it was casuing issues with code that no longer exists, ill change it back

Reply:
if its causing problems, then maybe it can be removed, i just want to make sure there is a reason